src/pages/Jokes.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { Card, CardTitle, Button, Col, CardText } from "reactstrap"
+import { useState, useEffect } from 'react'
+
+
+const DadJoke = () => {
+  const [joke, setJoke] = useState([])
+
+  const FetchJokes = async() => {
+    const res = await fetch("https://api.chucknorris.io/jokes/random");
```

Comment:
I see you went a different route with the API. While it's not a deal breaker, I would usually use camelCasing for functions that are within a component and reserve PascalCasing for component names.